.github/actions/environment/action.yml

```diff
@@ -82,5 +82,6 @@ runs:
         fi
         # Found that actions/checkout does not remove all temporary
         # files from previous runs, to avoid of it need to run:
+        git clean -xffd
```

Comment:
```markdown
# Whether to execute `git clean -ffdx && git reset --hard HEAD` before fetching
# Default: true
clean: ''
```

https://github.com/actions/checkout/blob/main/README.md

It seems, it should be performed by the checkout action.

Reply:
Yes, now I see it. But I think it is not bad that we also explicitly run this command.

Reply:
I don't mind anyway.